shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -388,6 +388,12 @@ void onStart() {
     Log.v(TAG, "onStart()");
     ensureAlive();
     doInitialFlutterViewRun();
+    // This is a workaround for a bug on some OnePlus phones. The visibility of the application
+    // window is still true after locking the screen on some OnePlus phones, and shows a black
+    // screen when unlocked. We can work around this by changing the visibility of FlutterView in
+    // onStart and onStop.
+    // See https://github.com/flutter/flutter/issues/93276
+    flutterView.setVisibility(View.VISIBLE);
```

Comment:
I meet an issue because of this edit. My use case is I use FlutterFragment as a tab in Activity, when use switched to  other tabs, I will hide the FlutterFragment. After this commit, every time Activity resumed, the flutterView will be foreced to visible, even the FlutterFragment is set to hide.

Reply:
Please file a new issue with minimum reproducible code, thank you.

Reply:
> Please file a new issue with minimum reproducible code, thank you.

this merge doest fix the issue;

this is my [code](https://github.com/chdo002/float_fragment.git) to reproduce,